v5/v5_with_library/v5_stable.ino

```diff
@@ -166,7 +173,7 @@ String BLE_Output = "";
 // bool batteriesCharged = false;
 
 String ret_string = "";
-
+String OWNER_NAME = "M Hamid Jamil";
```

Comment:
I think it should be moved to arduino_secret.h file, what do you think?

Reply:
Yes, you are right done.